hack/deployer/runner/ocp3.go

```diff
@@ -1,196 +0,0 @@
-// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I created https://github.com/elastic/cloud-on-k8s/issues/6441 to properly clean ocp3 in a follow-up.